<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTimeZone</class>
 <widget class="QDialog" name="DlgTimeZone">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>161</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,3,1" columnminimumwidth="0,0,0">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="labelHeader">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelPreselected">
     <property name="text">
      <string>Preselected</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KComboBox" name="cmbPreselected"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btnShowAll">
     <property name="text">
      <string>Show all</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelAll">
     <property name="text">
      <string>All timezones</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KComboBox" name="cmbAll"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btnShowPreselected">
     <property name="text">
      <string>Show preselected</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>452</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgTimeZone</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgTimeZone</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
